wordle-web/src/views/LeaderboardView.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div>
+    <v-card>
+      <h1>Leaderboard</h1>
```

Comment:
I'm not sure where it is or how to fix it (I've this problem in my deployment as well), but the leaderboard and wordle pages really don't like f5.

Reply:
the secret lies in your /public/staticwebapp.config.json file!

{
    "navigationFallback": {
        "rewrite": "/index.html"
    }, 
    "mimeTypes":{
        ".json": "text/json"
    }
}